src/main/scala/sbt/errorssummary/ConciseReporter.scala

```diff
@@ -152,15 +157,25 @@ private class ConciseReporter(logger: Logger,
     val noString = Option.empty[String]
     val (position, lineContent, pointer, prefix) =
       file.fold((noString, noString, noString, "")) { f =>
-        val showLine =
-          line.fold("")(l => s"${colored(colorFor(problem), l.toString)}:")
-        val showCol =
-          if (config.columnNumbers) offset.fold("")(c => s"${c + 1}:") else ""
-        val position =
-          s"""${colored(config.sourcePathColor, f)}:$showLine$showCol"""
-        val lineContent = Some(problem.position.lineContent).filter(_.nonEmpty)
-        val pointer     = problem.position.pointerSpace.map(sp => s"$sp^")
-        val prefix      = s"${extraSpace(problem.severity)}[${problem.id}] "
+        val lineCol =
+          "%s%s".format(line.fold("") { l =>
+            colored(colorFor(problem), "L%d:".format(l))
+          }, offset.filter(_ => config.columnNumbers).fold("") { c =>
+            "C%d:".format(c + 1)
+          })
```

Comment:
Any reason to prefer `format` over string interpolation? I would prefer to keep string interpolation if you don't mind.